EIPS/eip-9042.md

```diff
@@ -0,0 +1,94 @@
+---
+eip: 9042
```

Comment:
@abcoathup I used the PR number, but I guess I should be assigned another number ?

Reply:
I've assigned a number